### PR TITLE
Clarify what elements() does

### PR DESCRIPTION
**Pull request by r-barnes**

### CODE REVIEW

File: include/af/array.h [hunk 1]
```diff
@@ -572,7 +572,7 @@ namespace af
         af_array get() const;
 
         /**
-           get the number of elements in array
+           Get the number of elements in array. This is the product of the array's dimensions.
```
Comment (9prady9) [inline include/af/array.h]: perhaps "Get the total number of elements across all dimensions of the array" is more succinct ?

And move the next statement as a note `\note This is the product of the array's dimensions` - Although I think this is not required given that is very basic thing with multidimensional arrays, especially if we rephrase it to the above.

Comment (r-barnes) [reply include/af/array.h]: Fixed.

File: include/af/array.h [hunk 2]
```diff
@@ -1631,7 +1631,8 @@ extern "C" {
         @{
     */
     /**
-        \brief Gets the number of elements in an array.
+        \brief Gets the number of elements in an array. This is the product
+        of the array's dimensions.
```
Comment (9prady9) [inline include/af/array.h]: @r-barnes You forgot to change this one.

Comment (r-barnes) [reply include/af/array.h]: Fixed.

**Comment by 9prady9** [comment]
Thank you!
